Vagrantfile

```diff
@@ -50,7 +50,11 @@ Vagrant.configure("2") do |config|
     # The time in seconds to wait for the virtual machine to report an IP address
     v.ip_address_timeout = 130
     # Use differencing disk instead of cloning whole VHD
-    v.differencing_disk = true
```

Comment:
When did we add the bit about differencing? Linked Clone is all I've used... hmm

Reply:
Nevermind, this is HyperV - good to see they are making the naming consistent (as they should be).